dxr/plugins/rust/tests/test_cargo/dxr.config

```diff
@@ -0,0 +1,11 @@
+[DXR]
+enabled_plugins     = pygmentize rust
+disabled_plugins    = clang
```

Comment:
No need for this; it's off already since you explicitly listed what you want turned on.